cmd/peer-rest-common.go

```diff
@@ -18,68 +18,29 @@
 	peerRESTPrefix        = minioReservedBucketPath + "/peer"
 	peerRESTPath          = peerRESTPrefix + peerRESTVersionPrefix
 )
 
 const (
 	peerRESTMethodHealth                      = "/health"
-	peerRESTMethodServerInfo                  = "/serverinfo"
-	peerRESTMethodLocalStorageInfo            = "/localstorageinfo"
-	peerRESTMethodCPUInfo                     = "/cpuinfo"
-	peerRESTMethodDiskHwInfo                  = "/diskhwinfo"
-	peerRESTMethodNetHwInfo                   = "/nethwinfo"
-	peerRESTMethodOsInfo                      = "/osinfo"
-	peerRESTMethodMemInfo                     = "/meminfo"
-	peerRESTMethodProcInfo                    = "/procinfo"
-	peerRESTMethodSysErrors                   = "/syserrors"
-	peerRESTMethodSysServices                 = "/sysservices"
-	peerRESTMethodSysConfig                   = "/sysconfig"
-	peerRESTMethodDeleteBucketMetadata        = "/deletebucketmetadata"
-	peerRESTMethodLoadBucketMetadata          = "/loadbucketmetadata"
-	peerRESTMethodGetBucketStats              = "/getbucketstats"
-	peerRESTMethodGetAllBucketStats           = "/getallbucketstats"
 	peerRESTMethodVerifyBinary                = "/verifybinary"
 	peerRESTMethodCommitBinary                = "/commitbinary"
 	peerRESTMethodSignalService               = "/signalservice"
 	peerRESTMethodBackgroundHealStatus        = "/backgroundhealstatus"
 	peerRESTMethodGetLocks                    = "/getlocks"
-	peerRESTMethodLoadUser                    = "/loaduser"
-	peerRESTMethodLoadServiceAccount          = "/loadserviceaccount"
-	peerRESTMethodDeleteUser                  = "/deleteuser"
-	peerRESTMethodDeleteServiceAccount        = "/deleteserviceaccount"
-	peerRESTMethodLoadPolicy                  = "/loadpolicy"
-	peerRESTMethodLoadPolicyMapping           = "/loadpolicymapping"
-	peerRESTMethodDeletePolicy                = "/deletepolicy"
-	peerRESTMethodLoadGroup                   = "/loadgroup"
 	peerRESTMethodStartProfiling              = "/startprofiling"
 	peerRESTMethodDownloadProfilingData       = "/downloadprofilingdata"
-	peerRESTMethodCycleBloom                  = "/cyclebloom"
-	peerRESTMethodTrace                       = "/trace"
-	peerRESTMethodListen                      = "/listen"
-	peerRESTMethodLog                         = "/log"
-	peerRESTMethodGetLocalDiskIDs             = "/getlocaldiskids"
 	peerRESTMethodGetBandwidth                = "/bandwidth"
-	peerRESTMethodGetMetacacheListing         = "/getmetacache"
-	peerRESTMethodUpdateMetacacheListing      = "/updatemetacache"
-	peerRESTMethodGetPeerMetrics              = "/peermetrics"
-	peerRESTMethodGetPeerBucketMetrics        = "/peerbucketmetrics"
-	peerRESTMethodLoadTransitionTierConfig    = "/loadtransitiontierconfig"
 	peerRESTMethodSpeedTest                   = "/speedtest"
 	peerRESTMethodDriveSpeedTest              = "/drivespeedtest"
 	peerRESTMethodReloadSiteReplicationConfig = "/reloadsitereplicationconfig"
```

Comment:
and these as well, they do not send any data.

Reply:
I was hesitant for the speedtests.. but I guess the information only ones would be fine.